galaxy/tools/deps/mulled/invfile.lua

```diff
@@ -40,19 +40,47 @@ for i = 1, #test_binds_table do
     table.insert(test_bind_args, test_binds_table[i])
 end
 
+local conda_image = VAR.CONDA_IMAGE
+if conda_image == '' then
```

Comment:
I didn't - it was for if your playing around with the ``involucro`` command directly I guess. I can update this with an option at the mulled level if you'd like.